chat/src/main/java/greencity/controller/ChatController.java

```diff
@@ -305,38 +300,61 @@ public ResponseEntity<List<ChatRoomDto>> getGroupChats(Principal principal) {
     }
 
     /**
-     * Method for uploading an image.
+     * Method for uploading an images.
```

Comment:
Based on the method signature we can upload single image only
Please change `images -> image`